authorization/src/main/java/com/terabite/movement/controller/MovementController.java

```diff
@@ -53,6 +53,46 @@ public ResponseEntity<Movement> getMovementById(@PathVariable("id") long id)
         return new ResponseEntity<>(movement, HttpStatus.OK);
     }
 
+    @GetMapping("/movements/isolated")
```

Comment:
instead of hardcoding this, maybe this is a good place to use a Path parameter
`/type/{movement_type}` and then we would call v1/mov/type/isolated, and v1/mov/type/compound`

Hardcoding everything might make it a pain to change this in the future.